src/python/requirements.txt

```diff
@@ -24,3 +24,4 @@ PyJWT==1.5.0
 
 # service
 python-daemon==2.1.2
+setproctitle==1.1.10
```

Comment:
needs approval! @izoratti

Reply:
I don't this we should use this. Please note that we will remove the daemon run of Python: this has been a wrong implementation that we have carried on since we started the project and we must fix asap. As I explained during the scrum yesterday, FogLAMP will be governed by the foglamp bash script (in snap or in a standard installation), so the process will be spawned by the script when needed. I looked at the code, I don't think the changes applied and the use of the library is necessary (like here: https://github.com/foglamp/FogLAMP/blob/FOGL-517/src/python/foglamp/core/server_daemon.py#L95).

We should remove this.